ruby/lib/trailer_vote/media_types/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module TrailerVote
   module MediaTypes
-    VERSION = '0.13.1'
+    VERSION = '0.13.2'
```

Comment:
This breaks compatibility so should result in a major increase in the version number.